tests/examples/components/shared/test_aml_smpc.py

```diff
@@ -0,0 +1,115 @@
+import os
+import sys
+import unittest
+
+from cryptography.hazmat.primitives import serialization as crypto_serialization
+
+sys.path.append(
+    os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..", "..", ".."))
+)
+from examples.components.shared.aml_smpc import AMLSMPC
+
+
+class TestAMLSMPC(unittest.TestCase):
```

Comment:
Not testing big (or empty) messages in this one?

Reply:
I can add empty, for large nope, there is packet handling in AMLComm for large messages, there is no special treatment for them in this file.